clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -129,6 +129,8 @@
 
     private RequestFuture<Void> findCoordinatorFuture = null;
 
+    protected final String fencedInstanceErrorMessage = "Received fatal exception: group.instance.id {} get fenced in {}";
```

Comment:
nit: It's a bit odd to store this separately since the message is couple with the substitutions. Duplicating the message is not a big deal, but if you don't want to do that, I'd suggest just creating a method for the full log line.